### ISSUE TITLE
Can I run this bot on Linux?

### ISSUE THREAD

**Issue by Mkekala** [Maxx53/SCMBot#258]
I'm planning to make my BeagleBone Black a 24/7 steam market bot, but it can only run linux, not windows.

**Comment by Maxx53** [comment]
No, sorry. Look for more info https://github.com/Maxx53/SCMBot/issues?q=linux
